src/silx/gui/widgets/FrameBrowser.py

```diff
@@ -312,3 +312,100 @@ def setValue(self, value):
     def value(self):
         """Get selected value"""
         return self._slider.value()
+
+class FrameRateWidgetAction(qt.QWidgetAction):
+    def __init__(self, parent):
+        super().__init__(parent)
+        self._build()
+        
+    def _build(self):
+        widget = qt.QWidget()
+        layout = qt.QHBoxLayout()
+        widget.setLayout(layout)
+        self.line_edit = qt.QLineEdit()
+        layout.addWidget(qt.QLabel("FPS:"))
+        layout.addWidget(self.line_edit)
+        self.setDefaultWidget(widget)
+        
+class IntervalsWidgetAction(qt.QWidgetAction):
+    def __init__(self, parent):
+        super().__init__(parent)
+        self._build()
+        
+    def _build(self):
+        widget = qt.QWidget()
+        layout = qt.QHBoxLayout()
+        widget.setLayout(layout)
+        self.spinbox = qt.QSpinBox()
+        layout.addWidget(qt.QLabel("Interval:"))
+        layout.addWidget(self.spinbox)
+        self.setDefaultWidget(widget)
+
+class PlayButtonContextMenu(qt.QMenu):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._build()
+
+    def _build(self):
+        self.line_edit_action = FrameRateWidgetAction(self)
+        self.interval_action = IntervalsWidgetAction(self)
+        self.addAction(self.line_edit_action)
+        self.setFrameRate(value="10")
+        self.addAction(self.interval_action)
+        self.setInterval(value=1)
+        
+    def setFrameRate(self, value:str):
+        self.line_edit_action.line_edit.setText(value)
+        
+    def getFrameRate(self):
+        return int(self.line_edit_action.line_edit.text())
+    
+    def setInterval(self, value:int):
+        self.interval_action.spinbox.setValue(value)
+    
+    def getInterval(self):
+        return int(self.interval_action.spinbox.value())
+        
+class HorizontalSliderWithBrowserPlay(HorizontalSliderWithBrowser):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        
+        self.__timer = qt.QTimer(self)
+        self.__timer.timeout.connect(self._updateState)
+        
+        fontMetric = self.fontMetrics()
+        iconSize = qt.QSize(fontMetric.height(), fontMetric.height())
+        
+        self.playButton = qt.QPushButton(self)
+        self.playButton.setIcon(icons.getQIcon("camera"))
+        self.playButton.setIconSize(iconSize)
+        self.mainLayout.addWidget(self.playButton)
+        
+        self.playButton.clicked.connect(self._playStopSequence)
+        self.menu = PlayButtonContextMenu(self)
+        
+    def contextMenuEvent(self, event):
+        self.menu.exec_(self.mapToGlobal(event.pos()))
```

Comment:
Providing access to the settings this way raises 2 issues:
- The context menu is for the whole horizontal slider, not just the menu.
- There is no visual hints about the availability of those settings.

I would try to use of a QToolButton here which can display an arrow (see https://doc.qt.io/qt-6/qtoolbutton.html#popupMode-prop, https://github.com/silx-kit/silx/blob/main/src/silx/gui/plot/PlotToolButtons.py)